src/fallback.ts

```diff
@@ -323,7 +323,19 @@ export class GrpcClient {
       ) => {
         const [method, requestData, serviceCallback] = serviceStub[
           methodName
-        ].apply(serviceStub, [req, callback]);
+        ].apply(serviceStub, [
+          req,
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          (err: any, response: any) => {
```

Comment:
`err` is `Error|null`, `response` is probably an object (`{}`).  Let's avoid using `any`.

Reply:
SG. Done. Thanks